src/components/user/IFXUserDetail.vue

```diff
@@ -28,6 +28,12 @@ export default {
     IFXContactRoleDisplayEdit,
     IFXAffiliationRoleDisplayEdit,
   },
+  props: {
+    djangoEditOnly: {
+      type: Boolean,
+      default: false
```

Comment:
I usually include `required: false` if it isn't required but the Vue docs say that not required is the default.

Reply:
Done